### DISCUSSION TITLE
Welcome Email after registering

### DISCUSSION THREAD

**Discussion by epheterson** [mealie-recipes/mealie#1444] [Feature Request]
**Before submitting this feature request I have**
- [ X ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [ X ] Reviewed the existing feature requests and my feature has not been requested.
- [ X ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Sharing only an invite URL requires the user figure out what the base URL is on their own, and clicking the invite URL a second time brings up another registration page. It's confusing to send two separate links to a new user and explaining how to use them.

**(Optional): Suggest A Solution**
I'm not sure if this is coming with https://github.com/hay-kot/mealie/issues/833 (didn't see it mentioned), but it'd be great if users would get a simple email after signing up. The email should include a link to the base site, and their username.

An email clearly shows them where to go, gives them something to bookmark, and is a reference of when they signed up and the username they chose.

Doesn't need to be anything fancy, just some text and maybe the Mealie logo. It'd be nice if it matched the theme, but that's not necessary.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by RobCod** [comment]
+1
